### PR TITLE
[Feature-16608][Doc] Add Security Model for DS

### LINKED ISSUE #16608: [Feature] Security Model
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Add document to describes the Apache DolphinScheduler security model from the perspective of the user.  It is intended to help users understand the security model and make informed decisions about how to deploy and manage Apache DolphinScheduler. This document also describes the scope of security vulnerabilities for Apache DolphinScheduler. Developers are advised to read this article before reporting security vulnerabilities.

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wangxj3**
## Purpose of the pull request

close #16608

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by davidzollo — DISMISSED** [review]
+1
Great job
This is a very good beginning. We can gradually improve this security model and more suggestions are welcome.

**Comment by davidzollo** [comment]
By the way, please add a note in this following doc, the reporter should read our security model before submitting the security issues `https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/security.md`

**Comment by wangxj3** [comment]
> By the way, please add a note in this following doc, the reporter should read our security model before submitting the security issues `https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/security.md`

ok

File: docs/docs/en/security-model.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Apache DolphinScheduler Security Model
```
Comment (SbloodyS) [inline docs/docs/en/security-model.md]: It should be in the same directory as `security.md`.

Comment (wangxj3) [reply docs/docs/en/security-model.md]: security.md is the part of guide - security.
图片
```suggestion
# Apache DolphinScheduler Security Model
```
Is it inappropriate to put it here ?

Comment (SbloodyS) [reply docs/docs/en/security-model.md]: The `security.md` is not reflected in the document, which is a bug. I've create issue #16690 to fix it.

**Review by SbloodyS — APPROVED** [review]
+1

**Review by davidzollo — APPROVED** [review]
+1

**Pull request merged**